app/models/group_role_management/activation.rb

```diff
@@ -0,0 +1,40 @@
+# This is a PORO Model that encapsulates logic for modifying associations on Person/User models
+
+# Methods in this class are delegated to the Person model via the `activation` association.
+# This allows calls like `@person.activate!(:adopter)` to invoke logic defined here.
+#
+module GroupRoleManagement
```

Comment:
I was also thinking about how to define these methods, right now it takes a single argument that accounts for the group, or the name, how about splitting these into two separate possible arguments, group, name? I think that's more conventional. Then check at least one is provided.

Reply:
Nice.